src/node_jsvmapi.h

```diff
@@ -110,6 +110,7 @@ NODE_EXTERN napi_status napi_create_symbol(napi_env e,
 NODE_EXTERN napi_status napi_create_function(napi_env e,
                                              napi_callback cb,
                                              void* data,
+                                             napi_propertyname name,
```

Comment:
I think this should be `const char*` instead of `napi_propertyname`. The function name is not used as a property key.